vahti/parsers/tori/tori.py

```diff
@@ -50,11 +51,37 @@ def parse(self, html):
                 "link": f"https://www.tori.fi/vi/{item_id}.htm",
                 "seen": now,
             }
-            logger.debug(self.item_format(**new_item))
             result[item_id] = new_item
-
+        self.print_result(result,self.config)
```

Comment:
The printing functionality is in the `vahti.py` file. You should instead modify the `result`.